crates/rolldown/tests/rolldown/topics/hmr/add_watch_file/artifacts.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Assets
+
+## entry.js
+
+```js
+// HIDDEN [rolldown:runtime]
+// HIDDEN [rolldown:hmr]
+//#region entry.js
+var entry_exports = {};
+const entry_hot = __rolldown_runtime__.createModuleHotContext("entry.js");
+__rolldown_runtime__.registerModule("entry.js", { exports: entry_exports });
+const content = "input\n";
+console.log(content);
+
+//#endregion
+```
+# HMR Step 0
+
+
+## Meta
+
+- update type: full-reload
+- reason: no hmr boundary found for module `entry.js`
```

Comment:
I guess this line is expected? update propragted to `entry.js` which isn't a hmr boundary.